### ISSUE TITLE
Conflict with VS2 upon mixin load

### ISSUE THREAD

**Issue by LordDarthDan** [The-Fireplace-Minecraft-Mods/Annotated-DI#12]
### This Mod Version

4.0.1

### Other Mod Name

Valkyrien Skies 2

### Other Mod Version

2.1.0-beta.10

### What mod loader(s) are you seeing the problem on?

Forge

### Mod Loader Version

40.2.0

### Problem Description

After putting Annotated DI into my pack (due to adding Overlord), I've experienced a crash on the earliest stages of loading.
Crash report included:
https://mclo.gs/TUuhPfM

Correction: I've started this post as a CC: Tweaked (ver. 1.101.2) compatibility issue, but after I got to trying to reproduce it with a minimal setup, it turned out to actually be a Valkyrien Skies 2 issue:
https://mclo.gs/JgTFoPk

I have seen the other issue of this nature by one of the authors of VS, but I thought it might be helpful to point out that the issue still exists, and provide logs to what actually happens.

### Steps To Reproduce

1. Add the following mods:
    * Annotated DI
    * Kotlin for Forge
    * Valkyrien Skies 2
2. Launch the game
3. Experience a ResolutionException: Modules javax.inject and annotateddi export package javax.inject to module jackson.kotlin.dsl

### Crash Report or Error Logs

```shell
https://mclo.gs/TUuhPfM - with big pack

https://mclo.gs/JgTFoPk - minimal setup
```

**Comment by The-Fireplace** [comment]
Duplicate of #11, but I like the details here, so leaving this open for now.

**Comment by Waterpicker** [comment]
Fixed in 4.0.3
